tests/unit/NumericParserTest.php

```diff
@@ -70,6 +70,19 @@ public function itCanParseLessThan()
         $this->assertEquals(123, $parser->getValue());
     }
 
+    /** @test */
+    public function itCanNotParseStringValue()
```

Comment:
We need at least one test from using API, e.g for file size in Files tests to make sure that it works with URL parameters.

Reply:
It is in commit below.